recipes/spirv-tools/all/conanfile.py

```diff
@@ -43,8 +43,8 @@ def _compilers_minimum_version(self):
         return {
             "17": {
                 "apple-clang": "10",
-                "clang": "5",
-                "gcc": "7",
+                "clang": "7",
+                "gcc": "8",
```

Comment:
Is it no specific to this new version? Why would it raise for clang 5 and gcc 7 for previous versions?